istioctl/cmd/auth.go

```diff
@@ -121,6 +124,32 @@ THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 			return nil
 		},
 	}
+
+	validatorCmd = &cobra.Command{
+		Use:   "validate <policy-file1,policy-file2,...>",
+		Short: "Validate authentication and authorization policy",
+		Long: `Validate and find incorrectness in authentication and authorization policy files, such as:
+						* Referring to a non existing ServiceRole.
+						* ServiceRole not used.
+					`,
+		Example: "istioctl experimental auth validate policy1.yaml,policy2.yaml",
```

Comment:
I'm not sure if it's good to use comma to separate the files (compared to use spaces), is there any similar use cases in istioctl like this?

Reply:
Your example for `upgrade` using service files also does this. I think it's fine.

Also, iirc, if you use multiple arguments for the same flag, they have to be separated by commas with no spaces; otherwise either the first file get parsed or there will be some error, depends how you provide the files as command line arguments.

Reply:
Okay, Can you put the `-f` flag here?

Reply:
Done. Thanks!